projects/36/bhxiv/paper.md

```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
+Our achievements during the 2021 BioHackathon cover subjects related to mapping concepts across health data formats (standards and terminologies), federated ML and metadata for ML. First, we achieved our main goal to map concepts from the OHDSI OMOP to elements in the GA4GH Phenopacket schema. We mapped schemas, generated a database of synthetic health data represented in the OMOP CDM by means of Synthea and published our workflow on GitHub so it could be reused by the community, generated mapping queries in SQL, and created a prototype for a Phenopacket builder. Second, we stemmed work to map both standards to the Semantic Web, in particular we started modelling semantic Phenopackets incorporating SPHN requirements and working on the ETL transformation of OMOP to the semantic model of the rare disease patient registry data in EJP RD, which will enable mapping to semantic Phenopackets. Third, we started discussing FAIR federated ML and we detected common expectations and needs. We did this with the enriching experience of exchanging different strategies utilised among different academia and industry organisations. Last, we recognised the FAIRification of ML is indeed a challenging issue that requires a broader community discussion, involving several stakeholder representatives to facilitate agreements on standards such as the DOME recommendations or Schema.org, technologies such as OWL, infrastructure such as FAIR Data Points, and evaluation methods such as benchmarks, and arguably, more events like the Biohackathon to support all of these endeavors. It is worth highlighting that these outcomes are the seed for different collaborations.
 
```

Comment:
Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

Reply:
organizations --> organisations

Reply:
recognized --> recognised
By the way "endeavors" should be "endeavours"

Reply:
Thanks! I will change endeavours as well